NuKeeper.Abstractions/CollaborationPlatform/ICommitWorder.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace NuKeeper.Abstractions.CollaborationPlatform
+{
+    public interface ICommitWorder
+    {
+        string MakePullRequestTitle(IReadOnlyCollection<PackageUpdateSet> updates);
```

Comment:
I don't think we should use `PackageUpdateSet` that is very specific for Nuget, if we are going to support `Paket` this is going to be a problem

Reply:
That seems like a good way forward.

Reply:
Alright, I will try to get the concept finished this weekend.